sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -59,15 +59,14 @@ class ConfigurationSetting(Model):
     }
 
     kind = "Generic"
-    content_type = None
 
     def __init__(self, **kwargs: Any) -> None:
         super(ConfigurationSetting, self).__init__(**kwargs)
         self.key = kwargs.get("key", None)
         self.label = kwargs.get("label", None)
         self.value = kwargs.get("value", None)
         self.etag = kwargs.get("etag", None)
-        self.content_type = kwargs.get("content_type", self.content_type)
+        self.content_type = kwargs.get("content_type", None)
```

Comment:
Why defaults to None?

Reply:
I don't change the default value, the original value of self.content_type is None.